packages/matchbox/src/components/Tag/styles.js

```diff
@@ -0,0 +1,150 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const tagBase = props => `
+  display: inline-flex;
+  align-items: center;
+  white-space: nowrap;
+  padding: 1px ${tokens.spacing_300};
```

Comment:
It seems like this `1px` padding has caused some alignment issues in the app. Is it needed?

Reply:
Yeah it's required unfortunately to stick to the 4px baseline, let me see if there's another way to do this